pkg/pillar/base/logwrapper.go

```diff
@@ -0,0 +1,66 @@
+package base
+
+import "github.com/sirupsen/logrus"
```

Comment:
So the underlying logger is from Sirupsen, but for each microservice (incl. domainmgr) we turn it into LogObject: https://github.com/lf-edge/eve/blob/9.4/pkg/pillar/base/logobjecttypes.go#L271-L296
We add some additional JSON fields there:
```
fields["source"] = agentName
fields["pid"] = agentPid
```
This is useful when you need to filter logs belonging to a given microservice.
I wanted nettrace package to use LogObject of its caller (instead of plain Sirupsen logger), so that we can also filter them by "source" or "pid". At the same time I didn't want libs/nettrace to be dependent on pillar, so I made nettrace expect logger with the widely-used Sirupsen's APIs. And since LogObject has slightly different APIs, I had to add this adapter.

Reply:
Okay, got it, thanks!